lit_tests/kernel/wave/gemm.py

```diff
@@ -21,6 +21,8 @@
 BLOCK_N = tkl.sym.BLOCK_N
 BLOCK_K = tkl.sym.BLOCK_K
 BLOCK_B = tkl.sym.BLOCK_B
+LOAD_ELEMS_PER_THREAD = tkl.sym.LOAD_ELEMS_PER_THREAD
```

Comment:
`elements_per_thread` is now optional, does it still work without them?

Reply:
Yes, I can remove it.

I'm currently investigating the CI failure because I didn't see it initially in my test run, but I think I had different testing parameters.